lib/racker.rb

```diff
@@ -22,8 +22,8 @@ def response
     end
   end
 
-  def render
-    path = File.expand_path("../views/index.html.erb", __FILE__)
+  def render(template)
```

Comment:
so you made the possibility to use templates but still using those ifs in index view?